tests/test_on_data/test_imaging_interfaces.py

```diff
@@ -54,3 +57,25 @@ class TestSbxImagingInterface(ImagingExtractorInterfaceTestMixin, TestCase):
         dict(file_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "Scanbox" / f"sample.sbx")),
     ]
     save_directory = OUTPUT_PATH
+
+
+class TestBrukerTiffImagingInterface(ImagingExtractorInterfaceTestMixin, TestCase):
+    data_interface_cls = BrukerTiffImagingInterface
+    interface_kwargs = dict(
+        folder_path=str(
+            OPHYS_DATA_PATH / "imaging_datasets" / "BrukerTif" / "NCCR32_2023_02_20_Into_the_void_t_series_baseline-000"
+        )
+    )
+    save_directory = OUTPUT_PATH
+
+    def check_extracted_metadata(self, metadata: dict):
+        self.assertEqual(metadata["NWBFile"]["session_start_time"], datetime(2023, 2, 20, 15, 58, 25))
+        self.assertEqual(metadata["Ophys"]["Device"][0]["name"], "Bruker Fluorescence Microscope")
+        self.assertEqual(metadata["Ophys"]["Device"][0]["description"], "Version 5.6.64.400")
+        self.assertEqual(metadata["Ophys"]["ImagingPlane"][0]["imaging_rate"], 30.345939461428763)
+        self.assertEqual(metadata["Ophys"]["ImagingPlane"][0]["description"], "The plane imaged at 5e-06 meters depth.")
+        assert_array_equal(metadata["Ophys"]["ImagingPlane"][0]["grid_spacing"], [1.1078125e-06, 1.1078125e-06])
+        self.assertEqual(metadata["Ophys"]["TwoPhotonSeries"][0]["unit"], "px")
+        self.assertEqual(metadata["Ophys"]["TwoPhotonSeries"][0]["field_of_view"], [0.0005672, 0.0005672, 5e-06])
+        self.assertEqual(metadata["Ophys"]["TwoPhotonSeries"][0]["format"], "tiff")
+        self.assertEqual(metadata["Ophys"]["TwoPhotonSeries"][0]["scan_line_rate"], 15840.580398865815)
```

Comment:
This value seems absurdly high for a scan line rate, especially since the total imaging rate is average ~30 Hz

Could you find any official documentation about the `scanLinePeriod` field and its units?

Reply:
@CodyCBakerPhD 
"scanLinePeriod" is `6.3129e-05` which is the same as "framePeriod" (`0.032953338` in seconds) divided by "linesPerFrame" (`512`). The closest documentation that I found [here](https://bruker-control.readthedocs.io/_/downloads/en/latest/pdf/), if "scanLinePeriod" is also in seconds, then we get `0.063129 ` in ms which is very similar to these settings:
<img width="374" alt="Screenshot 2023-05-03 at 13 04 19" src="https://user-images.githubusercontent.com/24475788/235900685-3af084e4-2fc7-4680-9fdb-7b16c93cbc5f.png">
But taking `1 / 6.3129e-05` would produce this unrealistic value.
For `scan_line_rate`, could we use the imaging rate divided by the number of lines for each frame (`512`)? 
That would be `~ 0.059 Hz`

Reply:
In other words, there are a total of ~`30` frames per second, and `512` lines per frame, so a total of `15360` lines per second that need to be scanned